nautobot/extras/models/customfields.py

```diff
@@ -251,6 +309,12 @@ class CustomField(BaseModel, ChangeLoggedModel, NotesMixin):
         limit_choices_to=FeatureQuery("custom_fields"),
         help_text="The object(s) to which this field applies.",
     )
+    grouping = models.CharField(
+        max_length=50,
+        null=True,
```

Comment:
I think removing `null=True` will ensure that it defaults to "" on migration, and it is good practice not to set null=True on CharFields anyway.